plugins_src/autouv/wpc_autouv.erl

```diff
@@ -2212,62 +2487,128 @@ update_and_scale_chart(Vs0,We0) ->
 %%% Draw routines.
 %%%
 
-draw_background(#st{bb=#uvstate{bg_img=Image}}) ->
+draw_background(#st{bb=#uvstate{bg_img=Image, tile={U0,V0}, st=#st{shapes=Shs},id=Id}}) ->
     gl:pushAttrib(?GL_ALL_ATTRIB_BITS),
-    wings_view:load_matrices(false),
-
+    Matrices = wings_u:get_matrices(Id, original),
+    [{X0,_,_},{X1,_,_}] = obj_to_screen(Matrices, [{0.0,0.0,0.0},{1.0,0.0,0.0}]),
+    TileWidth = X1-X0,
+    TxSetMode = wings_wm:get_prop(wings_wm:this(), texture_set_mode),
+    if (TxSetMode) ->
+        Bin = << <<V:?F32,(?TILE_ROWS*1.0):?F32, V:?F32,0.0:?F32, (?TILE_ROWS*1.0):?F32,V:?F32, 0.0:?F32,V:?F32>>
+                 || V <- lists:seq(0,?TILE_ROWS) >>;
+    true ->
+        Bin = << <<V:?F32,20.0:?F32, V:?F32,-20.0:?F32, 20.0:?F32,V:?F32, -20.0:?F32,V:?F32>>
+                 || V <- lists:seq(-20,20) >>
+    end,
```

Comment:
indentation

Reply:
Dan, what is wrong with this indentation?
In the source I've been using spaces (no tab) and it looks OK for me even in the git viewer:
![image](https://github.com/user-attachments/assets/744b7447-f696-4ae6-a608-ef1ada29cc3b)